zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/BackgroundTaskManagerFactory.java

```diff
@@ -72,13 +74,15 @@ public BackgroundTaskManager build() {
 
   private List<Runnable> buildTasks() {
     final List<Runnable> tasks = new ArrayList<>();
-    int threadCount = 0;
+    int threadCount = 1;
+
+    tasks.add(buildIncidentMarkerTask());
 
     if (config.getArchiver().isRolloverEnabled()) {
-      threadCount = 1;
+      threadCount = 2;
       tasks.add(buildProcessInstanceArchiverJob());
       if (partitionId == START_PARTITION_ID) {
-        threadCount = 2;
+        threadCount = 3;
```

Comment:
:thinking: Arguably we could also just have 2, since the archiver jobs are fully asynchronous, so they could theoretically run by themselves on a single thread.

Reply:
Not sure if it's worth the risk :grin: